### ISSUE TITLE
Better Multiple Artists Support (FLAC)

### ISSUE THREAD

**Issue by lvitalis** [MoleMan1024/audiowagon#23]
As a music lover and obsessive audio library tagger,
I want AudioWagon to handle multiple artist tags,
so that I can browse my catalog easily.

**Use case**
Two famous artists, John and Bob, have released an album together, The John & Bob Duo Album.

The tracks are tagged like this :
- `ARTIST` contains "John"
- A second `ARTIST` tag contains "Bob"
- `ALBUM` contains The John & Bob Duo Album

**Result (in v0.3.4)
Only the first `ARTIST` tag encountered is listed in the Artists tab.
The second (and others) are ignored.

**Expected Result**
All artists should be present in the Artist tab.
The album should be available in all artists pages.

When clicking on John :
- John's White Album
- John's Second Album
- The John & Bob Duo Album

When clicking on Bob : 
- The John & Bob Duo Album
- Bob's Western Classics

[FLAC-multiple artists.zip](https://github.com/MoleMan1024/audiowagon/files/7198313/FLAC-multiple.artists.zip)

**Comment by MoleMan1024** [comment]
Was not aware that you can have multiple artist fields in metadata. Might be specific to FLAC, I mostly looked at .mp3s so far.

**Comment by rogerjacobsen** [comment]
Allthough this may be useful, I discovered that AW read and sort albums by Artist and not Album Artist. This results in a very crowded album list where the same album can appear tens of times in stead of only one time. This should be addresse in a future update. Seen in version 1.0.1.

**Comment by lvitalis** [comment]
Yes, for now the solution is to re-tag all compilations albums by entering "Various" in the "Artist" tag of all songs.

**Comment by MoleMan1024** [comment]
@rogerjacobsen : that will be adressed in #22 soon
